apps/web/src/services/alert-manager/v2/components/ServiceDetailTabsSettingsEventRuleFormCard.vue

```diff
@@ -205,6 +206,8 @@ const { mutate: eventRuleMutation, isPending: eventRuleMutationPending } = useMu
                 webhookId: data.webhook_id || 'global',
                 eventRuleId: data.event_rule_id,
             });
+        } else {
+            queryClient.invalidateQueries({ queryKey: eventRuleQueryKey });
```

Comment:
nitpick: maybe this invalidation is not working
```suggestion
queryClient.invalidateQueries({ queryKey: eventRuleQueryKey.value });
```